package.json

```diff
@@ -36,12 +36,12 @@
     "babel-preset-stage-0": "^6.0.15",
     "eslint": "^1.10.3",
     "eslint-plugin-react": "^3.6.2",
-    "react-hot-loader": "^1.3.0",
+    "react-hot-loader": "^3.0.0-beta.1",
```

Comment:
this should be moved from dev- to just dependencies, right?  since it's now used directly by the app and has it's own dev/prod code.